fh-dataman.json

```diff
@@ -0,0 +1,238 @@
+                  {
+                    "name": "FH_CLUSTER",
+                    "value": "${FH_CLUSTER}"
+                  },
+                  {
+                    "name": "FH_MBAAS_URL",
+                    "value": "${FH_MBAAS_URL}"
+                  },
+                  {
+                    "name": "FH_MBAAS_PASSWORD",
+                    "value": "${FH_MBAAS_URL}"
+                  },
+                  {
+                    "name": "FH_MBAAS_USERNAME",
+                    "value": "${FH_MBAAS_URL}"
+                  },
+                  {
+                    "name": "FHMBAAS_SERVICE_KEY",
+                    "value": "${FHMBAAS_SERVICE_KEY}"
+                  },
+                  {
+                    "name": "FH_LOG_LEVEL",
+                    "value": "${FH_DEFAULT_LOG_LEVEL}"
+                  },
+                ],
+                "resources": {
+                  "limits": {
+                    "cpu": "800m",
+                    "memory": "800Mi"
+                  },
+                  "requests": {
+                    "cpu": "200m",
+                    "memory": "200Mi"
+                  }
+                },
+                "imagePullPolicy": "IfNotPresent"
+              }
+            ],
+            "volumes": [
+              {
+                "name": "node-proxy",
+                "configMap": {
+                  "name": "node-proxy"
+                }
+              }
+            ]
+          }
+        }
+      }
+    },
+    {
+      "kind": "Route",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "fh-dataman",
+        "annotations": {
+          "openshift.io/host.generated": "false"
+        }
+      },
+      "spec": {
+        "host": "${FH_DATAMAN_ROUTER_DNS}",
+        "to": {
+          "kind": "Service",
+          "name": "fh-dataman-service"
+        },
+        "tls": {
+          "termination": "edge",
+          "insecureEdgeTerminationPolicy": "Allow"
+        }
+      }
+    }
+  ],
+  "parameters": [
+    {
+      "name": "FH_CLUSTER",
+      "value": "development",
+      "description": "the name of the cluster the fh-dataman was created from"
+    },
+    {
+    "name": "FH_MBAAS_URL",
+    "value": "${FH_MBAAS_URL}",
+    "description": "the url to reach fh-mbaas"
+    },
+    {
+    "name": "FH_MBAAS_PASSWORD",
+    "value": "${FH_MBAAS_PASSWORD}",
+    "description": "the password for fh-mbaas"
+    },
+    {
+    "name": "FH_MBAAS_USERNAME",
+    "value": "${FH_MBAAS_USERNAME}",
+    "description": "the username for fh-mbaas"
+    },
+    {
+      "name": "FH_DATAMAN_PORT",
+      "value": 8822,
+      "description": "The port the container is available at"
+    },
+    {
+      "name": "FH_DATAMAN_ENVIRONMENT",
+      "value": "production",
+      "description": "The environment"
+    },
+    {
+      "name": "FHMBAAS_SERVICE_KEY",
+      "description": "The MBaaS key",
+      "generate": "expression",
+      "from": "[a-zA-Z0-9]{40}"
```

Comment:
This shouldnt be generated here, it should be moved to a configmap. Can be removed for now and read from the fh-mbaas environment